decidim-core/db/migrate/20170713131308_migrate_user_roles_to_participatory_process_roles.rb

```diff
@@ -8,7 +8,7 @@ def up
       processes = participatory_processes.select { |process| process.organization == user.organization }
       processes.each do |process|
         user.roles.each do |role|
-          Decidim::ParticipatoryProcessUserRole.find_or_create_by(user: user, process: process, role: role)
+          Decidim::ParticipatoryProcessUserRole.find_or_create_by(user: user, participatory_process: process, role: role)
```

Comment:
I still believe migrations shouldn't ever, ever touch application code but do it directly via SQL. Otherwise applications that are long behind might have issues, as their application code would greatly differ from the database's state.